core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -431,41 +431,48 @@ public void importData() throws IOException, DaoException {
             }
         }
 
-                for (MutationEvent event : newEvents) {
-                    try {
-                        DaoMutation.addMutationEvent(event);
-                    } catch (DaoException ex) {
-                        throw ex;
-                    }
-                }
+        for (MutationEvent event : newEvents) {
+            try {
+                DaoMutation.addMutationEvent(event);
+            } catch (DaoException ex) {
+                throw ex;
+            }
+        }
 
-                for (ExtendedMutation mutation : mutations.values()) {
-                    try {
-                        DaoMutation.addMutation(mutation,false);
-                    } catch (DaoException ex) {
-                        throw ex;
-                    }
-                }
+        for (ExtendedMutation mutation : mutations.values()) {
+            try {
+                DaoMutation.addMutation(mutation,false);
+            } catch (DaoException ex) {
+                throw ex;
+            }
+        }
+
+        ClinicalAttribute attr = new ClinicalAttribute("MUTATION_COUNT", "Mutation Count", "Mutation Count", "NUMBER",
+            false, "1", geneticProfile.getCancerStudyId());
+        DaoClinicalAttributeMeta.addDatum(attr);
+        
+        Map<Integer, String> mutationCounts = DaoMutation.getMutationCounts(geneticProfileId);
+        for (Map.Entry<Integer, String> mutationCount : mutationCounts.entrySet()) {
+            DaoClinicalData.addSampleDatum(mutationCount.getKey(), "MUTATION_COUNT", mutationCount.getValue());
```

Comment:
Maybe create a function `DaoMutation.createMutationCountClinicalData(geneticProfileId)` (similar to `DaoMutation.calculateMutationCountByKeyword(geneticProfileId)`) and moved lines 450-457 there? It maybe also good to make `DaoMutation.getMutationCounts` private or even move the code inside `createMutationCountClinicalData`.